lib/spree_product_zoom/factories.rb

```diff
@@ -0,0 +1,45 @@
+FactoryGirl.define do
```

Comment:
FYI in your spec helper your can require spree's factories:
https://github.com/spree-contrib/spree_price_books/blob/master/spec/spec_helper.rb#L28